RNTester/Podfile

```diff
@@ -98,8 +98,38 @@ target 'RNTester-macOSIntegrationTests' do
   pod 'React-RCTTest', :path => "RCTTest"
 end
 
+# [TODO(macOS GH#217): these are special targets used by the internal Microsoft build pipeline
+target 'iosDeviceBuild' do
+  platform :ios, '9.0'
+  pods()
+end
+
+target 'iosSimulatorBuild' do
+  platform :ios, '9.0'
+  pods()
+end
+
+target 'macOSBuild' do
+  platform :osx, '10.14'
+  pods()
+end
+# ]TODO(macOS GH#217)
+
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     puts target.name
+    # [TODO(macOS GH#217): the internal Microsoft build pipeline needs iOS arm64e slices
+    if target.platform_name == :ios
+      target.build_configurations.each do |config|
+        archs = config.build_settings['ARCHS'];
+        if archs.nil?
+          archs = ['$(ARCHS_STANDARD)']
+        end
+        archs.push('arm64e');
+        config.build_settings['ARCHS'] = archs;
```

Comment:
Just a Ruby FYI–_and this is untested_–all of this can probably be replaced by:

```ruby
archs = config.build_settings['ARCHS'] ||= ['$(ARCHS_STANDARD)']
archs.push('arm64e')
```

or

```ruby
(config.build_settings['ARCHS'] ||= ['$(ARCHS_STANDARD)']) << 'arm64e'
```